doc/release-notes.md

```diff
@@ -274,6 +277,12 @@ To enable block pruning set `prune=<N>` on the command line or in
 `bitcoin.conf`, where `N` is the number of MiB to allot for
 raw block & undo data.
 
+A value of 0 disables pruning. The minimal value above 0 is 550. Your
+wallet is as secure with high values as it is with low ones. Higher
+values merely reduce the network traffic in case of reorganization of
```

Comment:
I agree this misunderstands things a bit. The minimum value is chosen to handle any realistic reorganization. Larger reorgs will be disastrous for many other reasons, so there's no need to even consider them here.

If you want to link to older release notes the best way, I think, would be to link to the historical release notes in master: https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.11.0.md#block-file-pruning . 
Alternatively you can link to the current release notes at the time of the v0.11.0 tag: 
https://github.com/bitcoin/bitcoin/blob/v0.11.0/doc/release-notes.md#block-file-pruning
These are probably the same but may potentially be missing some post-facto editorial fixes.